.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ on:
     # Run for pushes to master or a version branch, e.g. a PR was merged ...
     branches:
       - master
+      - test
```

Comment:
Unrelated change.

Reply:
```suggestion
```

Reply:
Ah... that bit, wasn't intended for this PR, mostly just for testingstuff my end. How should I sort this out?

Reply:
Delete the line.